app/providers/repositories/conversation/index.js

```diff
@@ -10,12 +10,16 @@ class ConversationRepository extends BaseRepository {
     return await super.prepareParams(params)
   }
 
+  async findExistedPrivateConversationByCid(cid) {
+    return this.findOne({ _id: cid })
+  }
+
   async findExistedPrivateConversation(ownerId, opponentId, isEncrypted) {
     ownerId = this.castObjectId(ownerId)
     opponentId = this.castObjectId(opponentId)
 
     const query = { type: "u" }
-    isEncrypted && (query["is_encrypted"] = isEncrypted)
+    query["is_encrypted"] = isEncrypted || { $ne: true }
```

Comment:
`$ne` and `$nin` are badly indexed in MongoDB. 
If you want to track the absence of the field, you could do the following:

```
db.collectionName.find({
  $or: [
    { is_encrypted: isEncrypted },      // Matches documents where the field is explicitly true/false
    { is_encrypted: { $exists: false } } // Matches documents where the field is absent
  ]
});
```